WTC_Flight/Src/main.c

```diff
@@ -286,6 +286,79 @@ int main(void) {
 
 	//HAL_NVIC_EnableIRQ(RTC_Alarm_IRQn);
 
+	////////////////
+	// BEGIN SD DEMO
+	//
+	// Demo of grabbing individual packets. Right now, splits packets on 'es'

```

Comment:
This is actually wrong, it doesn't split on `es`, it splits on `_.,_`, but I don't want to add another commit to this to fix a single non-essential line.